microsoft-edge/devtools-guide-chromium/javascript/reference.md

```diff
@@ -366,6 +366,41 @@ If you find yourself running the same debug code in the **Console** tool over an
 See [Run snippets of JavaScript on any webpage](./snippets.md).
 
 
+<!-- ====================================================================== -->
+## Use `# sourceURL` to name evaluated and inline code in the Sources tool
+
```

Comment:
```suggestion

```
the section never really explicitly directly says that the reason or effect of giving a (virtual) file name to block of code is, so that when you look at Sources > Nav pane > Files tab (or wherever in UI).  Specify exactly where in UI these virtual file names will appear.